src/index.ts

```diff
@@ -120,11 +121,16 @@ function getPins(config?: Array<number> | IConfig): Array<any> {
 }
 
 export class I2C extends Peripheral {
-
+  private busNumber: number;
   private devices: Array<I2cBus> = [];
 
   constructor(config?: Array<number> | IConfig) {
     super(getPins(config));
```

Comment:
There is a bit of a foundational issue with the constructor options I just realized that this PR exposes. The I2C bus and the pins need to be mapped 1 to 1 in order for this module to play nice with the rest of raspi.js.

So the first thing we need to do is figure out which pins map to which bus across all the different Raspberry Pi versions. Once we have this information, we'll need to add the appropriate `SDA1` and `SCL1` entries to https://github.com/nebrius/raspi-board/blob/master/src/index.ts.

Once that has been done, we'll need to update the constructor for this module to take a bus number _but not pins_ (i.e. removing the `pins` entry in the config object and reworking, if not removing, the `getPins` method).